rootfs/opt/entry-point.sh

```
@@ -3,10 +3,22 @@ set -eu
 
 EMPTY=""
 
+AUTH_TYPE=${AUTH_TYPE:-"md5"}
+
 if [ ! -f $USERLIST_PATH -a -n ${DRUPAL_DB_USER:-EMPTY} -a -n ${DRUPAL_DB_PASSWORD:-EMPTY} ] ; then
-  cat > $USERLIST_PATH <<EOF
-"$DRUPAL_DB_USER" "md5$(echo -n $DRUPAL_DB_PASSWORD$DRUPAL_DB_USER | md5sum - | cut -d" " -f1)"
-EOF
+  case $AUTH_TYPE in
+    md5)
+      credential="md5$(echo -n $DRUPAL_DB_PASSWORD$DRUPAL_DB_USER | md5sum - | cut -d" " -f1)"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)